### ISSUE TITLE
missing import for graphviz_layout

### ISSUE THREAD

**Issue by cthoyt** [networkx/networkx#2128]
When importing `networkx`, the functions from `networkx.drawing.nx_pydot` are not included in the `networkx` namespace. This causes the errors for the [example](http://networkx.readthedocs.io/en/networkx-1.10/reference/generated/networkx.drawing.nx_agraph.graphviz_layout.html#networkx.drawing.nx_agraph.graphviz_layout) and other use cases.

```python
>>> import networkx as nx
>>> G=nx.petersen_graph()
>>> pos=nx.graphviz_layout(G)
>>> pos=nx.graphviz_layout(G,prog='dot')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'networkx' has no attribute 'graphviz_layout'
```
**EDIT**: Additionally:

```python
>>> nx.draw_graphviz(G)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/site-packages/networkx/drawing/nx_pylab.py", line 984, in draw_graphviz
    pos = nx.drawing.graphviz_layout(G, prog)
AttributeError: module 'networkx.drawing' has no attribute 'graphviz_layout'
```

Here is the current `networkx/drawing/__init__.py`

```
# graph drawing and interface to graphviz

from .layout import *
from .nx_pylab import *
from . import nx_agraph
from . import nx_pydot
```

To fix this issue, there should be `from .nx_pydot import *` instead of `from . import nx_pydot` since these functions aren't being populated into the namespaces properly. Perhaps it's this way because it's not assumed that a user has graphviz and pydot/pygraphviz, so maybe a more clever use of imports could be the solution.

**Comment by hagberg** [comment]
This is intentional and was changed between networkx-1.10 and networkx-1.11.  
See for example the networkx-1.11 docs.   
http://networkx.readthedocs.io/en/networkx-1.11/reference/generated/networkx.drawing.nx_agraph.graphviz_layout.html#networkx.drawing.nx_agraph.graphviz_layout

Our previous clever use of imports was causing problems and confusion so we decided to require explicit imports to be clear on which version of the graphviz interface is selected.
